Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1913,13 +1918,4 @@ private extension Purchases {
         }
     }
 
-    private func postPaywallEventsIfNeeded() {
-        guard #available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *),
-              let manager = self.paywallEventsManager else { return }
-
-        self.operationDispatcher.dispatchOnWorkerThread(jitterableDelay: .long) {
-            _ = try? await manager.flushEvents(count: PaywallEventsManager.defaultEventFlushCount)
-        }
-    }
```

Comment:
Moved this method to `PurchasesOrchestrator` so it can be called after a successful purchase.